index.html

```diff
@@ -2532,183 +2532,169 @@ <h1>Servient Implementation</h1>
         <p> A <a>Servient</a> is a software stack that implements the
             WoT building blocks presented in the previous section. <a>Servients</a>
             can host and expose <a>Things</a> and/or consume <a>Things</a>.
-            Thus, <a>Servients</a> can perform in both the <a>WoT Server</a> and
-            <a>WoT Client</a> roles; the name (a portmanteau of server and client)
-            is based on this dual role.
+            Depending on the <a>Protocol Binding</a>, <a>Servients</a> can perform in both server and client role,
+            hence the portmanteau naming.
         </p>
-        <p> The previous sections and 
-            chapter <a href="#sec-wot-architecture"></a> 
-            describes how the WoT
-            building blocks conceptually relate to each other and to the
-            abstract architecture. When implementing these concepts, a
+        <p> The previous section describes how the WoT
+            building blocks conceptually relate to each other and how they correspond to the
+            abstract <a href="#sec-wot-architecture">Web Architecture</a>.
+            When implementing these concepts, a
             more detailed view is necessary that takes certain technical
             aspects into account. The detailed architecture of a <a>Servient</a>
-            implementation using the (optional) <a>WoT Scripting API</a> is
+            implementation using the (optional) <a>WoT Scripting API</a> building block is
             shown in <a href="#architecture-implementation"></a>.
-            This figure shows and implementation that is using the optional 
-            <a>WoT Scripting API</a>. 
-            Scripts are actually executed by a nested Scripting Runtime, 
-            while WoT-specific Servient Implementation aspects are managed
-            by the WoT Runtime. 
-            <a>Consumed Thing</a> and <a>Exposed Thing</a>
-            objects provide abstract interfaces to the Things
-            described in related Thing Descriptions. Scripts can
-            operate on Things using these abstractions by performing
-            abstract Interactions while the details of the Protocol Bindings are
-            managed automatically by the WoT Runtime.
+            Here, the <a>WoT Runtime</a> is also a <a>Scripting Runtime</a> system that
+            in addition to managing the WoT-specific aspects also interprets and executes the application scripts.
```

Comment:
It would be worth clarifying that it is also possible to implement a WoT Servient in other (non-scripting) languages, such as C, C++, Java, C#, ...

Reply:
That is the point of 8.10.1 and why we explicitly state this section explains the variant with the optional Scripting API.